MediaGalleryUi/Model/SearchCriteria/CollectionProcessor/FilterProcessor/ContentField.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Model\SearchCriteria\CollectionProcessor\FilterProcessor;
+
+use Magento\Framework\Api\Filter;
+use Magento\Framework\Api\SearchCriteria\CollectionProcessor\FilterProcessor\CustomFilterInterface;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\MediaContentApi\Api\GetAssetIdsByContentFieldInterface;
+
+/**
+ * Class responsible to filter a content field
+ */
+class ContentField implements CustomFilterInterface
+{
+    private const TABLE_ALIAS = 'main_table';
```

Comment:
A bit of inconsistency: there is a constant for the table name but no constant for `id` field name. I'd have both on none